install/k8s/README.md

```diff
@@ -6,8 +6,8 @@ Install kubernetes 1.4 or higher using http://kubernetes.io/docs/getting-started
 
 ### Install Contiv
 
-* Replace all instances of NETMASTER_IP in contiv/contiv.yaml with the master IP
-* Replace VLAN_IF with the data plane interface
+* Replace all instances of `__NETMASTER_IP__` in contiv/contiv.yaml with the master IP.
+* Replace `__VLAN_IF__` with the data plane interface. If there is only one interface set `__VLAN_IF__` to `""` or delete the VLAN_IF section.
 * Optional: Replace the contiv version(v0.1-11-30-2016.20-08-20.UTC) with the desired release/test version.
 * On the management node, run
```

Comment:
This needs to sound less prescriptive :-). Something like, on the management node, create a default network and default epg. Example follows:
...